<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #D0D0D0;</string>
  </property>
  <widget class="QPushButton" name="go_to_wallet_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>600</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font-family: 'Montserrat';
font-style: normal;
font-weight: 400;
font-size: 13px;
line-height: 16px;
color: #E4E4E4;
border: none;
text-align:center;
}
QPushButton:hover {
color:#fff
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_mark_field">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>245</width>
     <height>43</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background: #FFFFFF;
border-radius: 20%;
padding: 13px 17px;
font-family: 'Montserrat';
font-style: normal;
font-weight: 400;
font-size: 13px;
line-height: 16px;
}
QLineEdit:focus{
	border: 1px solid #000;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="change_mark_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>245</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	 background-color: #62B1FF;
	border-radius: 20%;
	font-family: 'Montserrat';
	font-style: normal;
	font-weight: 400;
	font-size: 15px;
	color: #000000;
}
QPushButton:hover{
   background-color:#7DBFFF;
}



</string>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_new_mark">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Montserrat';
font-style: normal;
font-weight: 700;
font-size: 35px;
line-height: 122px;
color: #000;</string>
   </property>
   <property name="text">
    <string>Новая оценка</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
